_posts/blog/2017-02-23-security-performance-implications.md

```diff
@@ -0,0 +1,158 @@
+
+The purpose of this two part series was to measure the performance impact of
+various security configurations on a cluster running Apache Accumulo’s
+continuous ingest suite. The tests were performed using Amazon Web
+Services (AWS), Hortonworks Data Platform 2.4 and Accumulo 1.7. Each of
+the five different security settings in Accumulo 1.7 was tested
+including: no security, SSL, Kerberos Auth, Kerberos Auth-Int, and
```

Comment:
Technically, it would be better to use "SASL with Kerberos authentication and quality of protection". Obviously, you don't need to include that mouthful on each list item.